src/main/java/MysqlDatabase.java

```diff
@@ -104,6 +104,31 @@ public boolean insertCommitToDatabase(CommitStructure commit) throws SQLExceptio
         return commits;
     }
 
+    /**
+     * Selects all rows from the table "commit" in the database.
+     * @return Returns a list with all of the rows read as CommitStructure objects
+     */
+    public CommitStructure selectSpecificRow(String commitID) throws SQLException {
+        CommitStructure commit = new CommitStructure();
+        if(connection!=null){
+            PreparedStatement preparedStatement = connection.prepareStatement("SELECT * FROM commit WHERE commitID='" + commitID + "'");
```

Comment:
Should probably use `?` symbol to prevent injection but no big deal for this assignment imo

Reply:
Oh yes you are right. Consistency is important :) will fix it